python/arc/autolinker/autolinker.py

```diff
@@ -913,7 +872,7 @@ def cluster_viewer(self):
     ## TODO: this method currently breaks Sphinx, need to investigate
     displayHTML(html2)
```

Comment:
@marcell-ferencz-databricks 
sphinx will run outside of databricks
you can get spark context and check clusterUsageTags, if that is set you are inside DBR and this should run, outside of DBR this will crash.

Reply:
happy to merge this PR with this TODO and we can fix this in a separate PR